config/locales/de.yml

```diff
@@ -904,6 +904,8 @@ de:
         reload: Zusammenfassung neu laden
         with_extra_charge: 'mit Aufschlag:'
         without_extra_charge: 'ohne Aufschlag:'
+      edit_note:
+        title: Bestellschein bearbeiten
```

Comment:
Why did you change it from "Notiz" to "Bestellschein"? IMHO it still refers to "Note"...

Reply:
you have to remember, i don't know german from dutch!  i didn't know what the original was.  :)  

will fix this eve

Reply:
AFAIK all untranslated texts are from benni and therefore in German langugage.